pyincore/analyses/waterfacilityrestoration/waterfacilityrestoration.py

```diff
@@ -48,33 +48,40 @@ def run(self):
         if pf_interval is None:
             pf_interval = 0.05
 
-        (inventory_restoration_map, pf_results, time_results) = self.waterfacility_restoration(inventory_list,
-                                                                                               mapping_set,
-                                                                                               restoration_key,
-                                                                                               end_time,
-                                                                                               time_interval,
-                                                                                               pf_interval)
+        restoration_table_dataset = self.get_input_dataset("restoration_table").get_csv_reader()
+        restoration_table = AnalysisUtil.get_csv_table_rows(restoration_table_dataset, ignore_first_row=False)
+
+        damage = self.get_input_dataset("damage").get_csv_reader()
+        damage_result = AnalysisUtil.get_csv_table_rows(damage, ignore_first_row=False)
+
+        (inventory_restoration_map, pf_results, time_results, func_results) = self.waterfacility_restoration(
+            inventory_list, damage_result, mapping_set, restoration_key, end_time, time_interval, pf_interval,
+            restoration_table)
 
```

Comment:
Is the format aligned with PEP8?

Reply:
I don't see any pep8 warnings and reformat code doesn't change anything about that line.